### ISSUE TITLE
Enhancement: Provide nanodet-m-x0.5 Pretrained Model

### ISSUE THREAD

**Issue by MarsTechHAN** [RangiLyu/nanodet#251]
Hi, I find that nanodet-m really friendly to low-end arm devices. I was wondering is that possibile to provide a even smaller x0.5 pretrained model to boost the speed for those low end devices?
Sincerely

**Comment by RangiLyu** [comment]
Model                 | Backbone           |Resolution|COCO mAP| FLOPS |Params | Pre-train weight |
:--------------------:|:------------------:|:--------:|:------:|:-----:|:-----:|:-----:|
NanoDet-m-0.5x        | ShuffleNetV2 0.5x  | 320*320  |  13.5  |  0.3G | 0.28M | [Download](https://drive.google.com/file/d/1rMHkD30jacjRpslmQja5jls86xd0YssR/view?usp=sharing) |
